docker-compose/merging/case-import-job/init-job-container.sh

```diff
@@ -3,7 +3,7 @@
 # Create log file for job execution
 touch /root/job-execution.log
 # Create env vars to access from crontab
-printenv | sed 's/^\(.*\)$/export \1/g' > /root/project_env.sh
+printenv | sed 's/^\(.*\)$/export \1/g' | sed -e 's/=\(.*\)/="\1"/' > /root/project_env.sh
```

Comment:
Why this diff ?

Reply:
printenv creates a file with e.g.

FOO=BAR

This file is sourced in the crontab before running the job. For strings with specials chars in them, it breaks. Our case:
SPRING_CONFIG_LOCATION=/foo;/bar;/baz

after sourcing, the env contains:
SPRING_CONFIG_LOCATION=/foo


the change makes it so that the fila has:
SPRING_CONFIG_LOCATION="/foo;/bar;/baz"

more robust handling could be handled (like escaping the quotes) but for now it's enough